blueocean-dashboard/src/main/js/components/testing/TestService.js

```diff
@@ -0,0 +1,57 @@
+import { BunkerService } from '../../../../../../blueocean-core-js/src/js/services/BunkerService';
```

Comment:
@cliffmeyers I think I need to publish corejs so I can just import in the line below?